pkg/sql/sem/tree/format.go

```diff
@@ -666,6 +666,14 @@ func (ctx *FmtCtx) CloseAndGetString() string {
 	return s
 }
 
+// CopyBytes returns the contents of the unread portion of the buffer by
+// creating a copy.
```

Comment:
I don't think you need to add this function here -- just copy bytes at the call site.